rules/framework.bzl

```diff
@@ -512,8 +512,28 @@ def _merge_root_infoplists(ctx):
 
     return output_plist
 
+def _attrs_for_split_slice(attrs_by_split_slices, split_slice_key):
+    if len(attrs_by_split_slices.keys()) == 0:
+        return []
+    elif len(attrs_by_split_slices.keys()) == 1:
+        return attrs_by_split_slices.values()[0]
+    else:
+        return attrs_by_split_slices[split_slice_key]
+
 def _apple_framework_packaging_impl(ctx):
-    framework_files = _get_framework_files(ctx)
+    # The current build architecture
+    arch = ctx.fragments.apple.single_arch_cpu
+
+    # The current Apple platform type, such as iOS, macOS, tvOS, or watchOS
+    platform = str(ctx.fragments.apple.single_arch_platform.platform_type)
+
+    # When building with multiple architectures (e.g., --ios_multi_cpus=x86_64,arm64), we should only pick the slice for the current configuration.
+    split_slice_key = "{}_{}".format(platform, arch)
```

Comment:
how abt `arm64`?

Reply:
If we build with --ios_multi_cpus=arm64, split_slice_key is ios_arm64. Is it what you're asking?

Reply:
ahh i see yup makes sense!